CRM/Utils/System/Base.php

```diff
@@ -132,7 +132,6 @@ public function url(
     $query = NULL,
     $absolute = FALSE,
     $fragment = NULL,
-    $htmlize = TRUE,
     $frontend = FALSE,
     $forceBackend = FALSE
   ) {
```

Comment:
@nganivet this signature change will break your standalone integration FYI.
I'm not sure how it's implemented but if the CRM_Utils_System_Standalone version of this function is reasonably simple then it might make sense to move it here (to the base class) instead of having this be a nullop.